app/services/sqs/publish_hearing.rb

```diff
@@ -138,12 +189,20 @@ def hearing_location(court_centre_id)
     end
 
     def session_hash
-      {
-        courtLocation: cjs_location,
-        dateOfHearing: defendant.dig(:offences, 0, :judicialResults, 0, :orderedDate),
-        postHearingCustody: PostHearingCustodyCalculator.call(offences: defendant[:offences]),
-        sessionValidateDate: defendant.dig(:offences, 0, :judicialResults, 0, :orderedDate),
-      }
+      if function_type == "OFFENCE"
```

Comment:
I wonder if it wouldn't be better to post court applications to its own SQS queue. Then we wouldn't need this `function_type` field.

Reply:
Hmm makes sense, @msahilan07 would you be open to MAAT API subscribing to an applications to court queue so that it can differentiate that way?